gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/04_add_theme_type.yml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.4.0_add_theme_type
+      author: GraviteeSource Team
+      changes:
+        - addColumn:
+            tableName: ${gravitee_prefix}themes
+            columns:
+              - column: { name: type, type: nvarchar(64), constraints: { nullable: false }, defaultValue: PORTAL }
```

Comment:
I think we had issue with adding column with 'nullable: false'.
If I remember will, it works with unit tests, but not with an existing db.
@ytvnr I think you had this issue.